channel_acceptor.go

```diff
@@ -0,0 +1,111 @@
+package lnd
```

Comment:
possible to move this out of the `lnd` package?

Reply:
Yup will do